x/interchainstaking/keeper/ibc_packet_handlers_test.go

```diff
@@ -1618,3 +1619,116 @@ func (s *KeeperTestSuite) Test_v046Callback() {
 	// assert that fee collector module balance is now 100denom more than before HandleMsgTransfer()
 	s.Require().Equal(feeMaccBalance2.AmountOf("denom").Sub(feeMaccBalance.AmountOf("denom")), sdk.NewInt(100))
 }
+
+func (s *KeeperTestSuite) Test_v045Callback_NilResp() {
+	s.SetupTest()
```

Comment:
logic is duplicated between both tests.  Let's make it a table-driven test to make things a bit simpler and clearer.

Reply:
check #316